cdragontoolbox/hashes.py

```diff
@@ -349,36 +336,26 @@ def substitute_region_lang(self):
             replacement = r'plugins/\1/%s/%s/' % region_lang
             self.check_iter(regex.sub(replacement, p) for p in known)
 
-    def substitute_basename_words(self, plugin=None, fileext=None, words=None, amount=1):
-        """Replaces {amount} side by side words in all basenames (default=1 for simple 1 word substitution).
+    def substitute_basename_words(self, plugin=None, fileext=None, words=None, amount=None, x=1, y=1):
+        """Substitutes {amount} side by side words in all basenames or replaces
+        x side by side words with y words (default=1 for simple 1 word substitution).
         Additionally, a plugin name and file extension can be specified to filter on."""
 
+        if amount:
+            x = amount
+            y = amount
```

Comment:
If there are only 2 values to configure, don't use 3 parameters, or properly check for conflicting parameters.

Reply:
Hm, do you think not having the ``amount`` parameter and just x and y is better? I thought having a parameter for both would be comfortable, but it might very well just be unnecessary.

Reply:
I thinks it's fine to only have `x` and `y`. But we should find better names for these parameters.
Something like `nsrc`/`ndst` or `nfrom`/`nto`.

Reply:
Not sure about the name. Tell me if you would prefer a different name.